etna/lib/etna/clients/metis/client.rb

```diff
@@ -24,8 +24,13 @@ def list_all_folders(list_all_folders_request = ListFoldersRequest.new)
       end
 
       def list_folder(list_folder_request = ListFolderRequest.new)
-        FoldersAndFilesResponse.new(
-          @etna_client.folder_list(list_folder_request.to_h))
+        if list_folder_request.folder_path != ''
```

Comment:
This also fixes a bug that hadn't been really tested -- 'listing' the root folder is actually backed by the bucket_list action, but code paths that accept arbitrary folder_path values won't see this on their own, so it is necessary to catch and handle it better here.